tests/test_cloud_row_push.py

```
@@ -0,0 +1,112 @@
+    monkeypatch.setenv(_cloud_sync.ENV_KEY, "fake-key")
+
+    with patch.object(_cloud_sync, "_post", return_value=1):
+        _cloud_sync.push(brain)
+
+    conn = sqlite3.connect(brain / "system.db")
+    row = conn.execute(
+        "SELECT last_push_at FROM sync_state WHERE brain_id = ?",
+        ("11111111-2222-3333-4444-555555555555",),
+    ).fetchone()
+    conn.close()
+    assert row and row[0]  # timestamp was recorded
```

Ruff (0.15.10)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)